rectools/metrics/classification.py

```diff
@@ -371,6 +371,27 @@ def _calc_per_user_from_confusion_df(self, confusion_df: pd.DataFrame, catalog:
         return mcc
 
 
+@attr.s
+class HitRate(SimpleClassificationMetric):
+    """
+    HitRate calculates the fraction of users for which the correct answer is included in the recommendation list
+
+    The HitRate equals to ``tp / (tp + fn)`` where
```

Comment:
Hi! Thank you for the PR
However this formula is for `Recall`, not for `HitRate`.

P.S. If your PR is ready, please add reviewers (me or @blondered). If not, please mark it as draft

Reply:
Sorry, when I read the artcile about metrics, the wrong calculation method stuck in my head.. Fixed

Reply:
That's ok

For future: please do not resolve the threads yourself, the thread must be resolved by its creator